.github/workflows/docker_publish.yml

```diff
@@ -57,17 +57,34 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Configure docker image tag and build arguments
+        run: |
+          image_base_tag=ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_version }}-${{ matrix.compiler}}-hdf5_${{ matrix.hdf5_version}}-moab_${{ matrix.moab_version }}
+          if [ ${{ matrix.geant4_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-geant4_${{ matrix.geant4_version }}
+          fi
+          if [ ${{ matrix.double_down_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-double_down_${{ matrix.double_down_version }}
+          fi
+          echo "image_base_tag=${image_base_tag}" >> "$GITHUB_ENV"
+
```

Comment:
Once again, if the geant and DD state is always in the tag name, we don't need this logic